src/Nelmio/Alice/Fixtures/Builder/Methods/ListName.php

```diff
@@ -25,21 +25,50 @@ class ListName implements MethodInterface
      */
     public function canBuild($name)
     {
-        return 1 === preg_match('#\{([^,]+(\s*,\s*[^,]+)*)\}#', $name, $this->matches);
+        if (1 !== preg_match('/\{(?<content>.*)\}/', $name, $this->matches)) {
```

Comment:
@theofidry There's no reason to use `$this->matches` here - that was being used when we were testing in the affirmative to avoid preg matching twice. Not my best code ;)